app/assets/javascripts/pages/agent-edit-page.js.coffee

```diff
@@ -13,6 +13,15 @@ class @AgentEditPage
           e.preventDefault()
           alert 'Sorry, there appears to be an error in your JSON input. Please fix it before continuing.'
 
+      if $(".link-region") && $(".link-region").data("can-receive-events") == false
```

Comment:
`$(".link-region")` will always be truthy in jQuery.  You want `if $(".link-region").length && ...`